ts/components/messages/MessageList.tsx

```diff
@@ -296,6 +301,15 @@ class MessageList extends React.Component<Props, State> {
     return this.state.itemLayouts[index];
   };
 
+  private onLongPress = (id: string) => {
```

Comment:
Ok, this is the same code I used to proof the feature, but I guess it is wrong 😢 
It is based on time, and you have no assurance about it will work on several devices in several different conditions.
For example, if the device that is running the app has a heavy background task, then 10 milliseconds are not enough to have the list's height refreshed.

I would propose another approach:
In **AnimatedFlatList** detected when [layout changes ](https://facebook.github.io/react-native/docs/view#onlayout)
Then check if the last _longpressed_ is the last in messages array (you could save the id in the state). If yes, then scroll to the last index. It should work because at this moment you are sure the list's height is already changed

Reply:
Done in commit 27a6e60